src/core/boot.js

```diff
@@ -32,6 +32,7 @@ module.exports = (self) => {
       (cb) => {
         self.log('initialized')
         self.state.initialized()
+        self.emit('init');
```

Comment:
This will failing linting (semicolons)

Reply:
Sorry about that. Linting already reports 87 problems for me in master so that one got lost in the noise.  Pushed a fix.